report_thesis/src/sections/background/linear_and_regularization_models/enet.tex

```diff
@@ -0,0 +1,15 @@
+\subsubsection{Elastic Net Regression (ENet)}
+Despite \gls{lasso} regression's ability to perform feature selection, it can struggle when the dataset contains highly correlated features.
```

Comment:
For LASSO we wrote that it works well in settings with multicollinearity and here we say that it does not work well with highly correlated features.
I know the distinction is that multicollinearity is collinearity at a sort of local scop and here we mean correlation at a more global scope. 
Do you think we should make that more obvious to the reader - in any case to clearly demonstrate that we know the difference.